eth-connector/src/deposit_event.rs

```diff
@@ -80,9 +79,10 @@ impl FtTransferMessageData {
         // The first data section should contain fee data.
         // Pay attention, that for compatibility reasons we used U256 type
         // it means 32 bytes for fee data
-        let mut data = U256::from(self.fee.as_u128()).as_byte_slice().to_vec();
+        let mut data = [0; 52];
```

Comment:
Is this code used just in the tests?
Also, why is the size of the data 52? The size of the recipient is dynamic and not predictable

Reply:
20 bytes, it is address on Aurora. This function used in `deposit` function in `EthConnector` contract.